dynawo/sources/Common/DYNGraph.cpp

```diff
@@ -118,14 +120,15 @@ Graph::findAllPaths(const unsigned int& vertexOrigin, const unsigned int& vertex
 
 bool
 Graph::findAllPaths(const unsigned int& vertexOrigin, const unsigned int& vertexExtremity,
-        PathDescription& currentPath, vector<bool> &encountered, list<PathDescription> &paths, FilteredBoostGraph & filteredGraph,
+        PathDescription& currentPath, boost::unordered_set<unsigned int>& encountered, list<PathDescription> &paths, FilteredBoostGraph & filteredGraph,
```

Comment:
should be const

Reply:
i am not sure what should be const with the comment could you do it ?